content/techniques/mongo.md

```diff
@@ -70,6 +70,24 @@ Alternatively, the `@Prop()` decorator accepts an options object argument ([read
 name: string;
 ```
 
+In case if you want to specify relation to another model, later for pupulating, you can use `@Prop()` decorator as well. For example, if `Cat` has `Owner` which is stored in different collection called `owners`, property should have type and ref. For example:
```

Comment:
Cats do not have owners, they have staff. 😺

Reply:
It can be staff or whatever, didn't find appropriate relation :)